### DISCUSSION TITLE
[Scam Alert] meta-force

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#21] [General]
Meta force
ادرس سایت : https://meta-force.space/

نوع کلاهبرداری : هشدار هرمی و پانزی

توضیحات سایت : فورس یک اکوسیستم ارز دیجیتال جهانی است که اعضای آن با استفاده از ابزارها و الگوریتم توزیع سود فوری در قراردادهای هوشمند به یکدیگر کمک می کنند تا به رفاه مالی برسند.
سیستم META FORCE متعلق به جامعه ما است و کاملاً غیرمتمرکز است، به این معنی که شفاف، ایمن و مقاوم در برابر تأثیرات خارجی است.
قرارداد هوشمند یک الگوریتم خود اجرا کننده (کد برنامه) است. شفافیت و امنیت اجرای منطق یا نقل و انتقالات در بلاک چین را تضمین می کند.
پلتفرم Force باز و در دسترس همه است. فرآیند ورود به جامعه و شروع به کسب سود به قدری ساده است که فقط به یک لپ تاپ یا گوشی هوشمند با اتصال به اینترنت نیاز دارید. پلتفرم‌های ماتریسی را توسعه دهید و شرکای خود را به ساختار دعوت کنید، که برای آن در ارزهای دیجیتال POLYGON پاداش دریافت می‌کنید.

بررسی های انجام شده :
این پلتفرم یکی از پلتفرم های دقیقا یکسان با forsage است و فرد سازنده معرفی شده Lado Okhotnikov در هر دو یکسان است .
بررسی forsage قبلا در لیست هشدار قرار داده شده است.

[https://github.com/Blockunity/WarningList/discussions/5](5)

همچنین فعالیت هر دو پلتفرم دقیقا عملکرد یکسانی دارد و به صورت ماتریکسی فعالیت میکنند و دقیقا همانند شبکه های هرمی در حال پیشرفت هستند این پلتفرم صرفا یک قرارداد هوشمند است و کاربرد ندارد و مشکلی را هم حل نمیکند . سایت در سال 2022 فعالیت خودش را اغاز کرده است.

![1](https://user-images.githubusercontent.com/103103225/184295153-8b79fe5d-3d70-4823-8bc9-c9a1619db8e8.jpg)
![4](https://user-images.githubusercontent.com/103103225/184295163-0c2427b6-b7bf-4b83-93cf-fd21e2028ebe.jpg)
![2](https://user-images.githubusercontent.com/103103225/184295157-1073c8c4-7ffe-48ac-a877-1fc89229e734.jpg)
![3](https://user-images.githubusercontent.com/103103225/184295160-6e5e6e77-bebf-4e1f-a993-9132b89bb12d.jpg)
![5](https://user-images.githubusercontent.com/103103225/184295164-c51649d1-e3ad-4c9e-9d46-4da70b0bed93.jpg)
![6](https://user-images.githubusercontent.com/103103225/184295167-6a2ab9bd-b01a-4958-bbbb-81fb0d63958c.jpg)

اطلاعات whois سایت : اطلاعات ثبت کننده دامنه محرمانه می باشد . سایت در سال 2022 فعالیت خودش را اغاز کرده است.

![7](https://user-images.githubusercontent.com/103103225/184295249-aa2644a7-1242-434b-a8f0-53dc4ba24479.jpg)

کانال های ایرانی زیادی در تلگرام در حال فعالیت و دامن زدن به این سبک هرمی قراردادهوشمندی هستند .

![8](https://user-images.githubusercontent.com/103103225/184295277-920a3908-29ea-4f13-b85b-7afac1d8e85a.jpg)

سایت خبری تحلیلی coindesk گزارشی کار کرده تحت عنوان (( چگونه forsage و metaforce و سایر طرح های هرمی قرارداد هوشمند را متوقف کنیم)) میتوانید از لینک زیر مطالعه کنید.
https://www.coindesk.com/layer2/2022/08/08/how-to-stop-forsage-meta-force-and-other-smart-contract-pyramid-schemes/

![9](https://user-images.githubusercontent.com/103103225/184295304-a8443b11-201e-4555-88e2-5c5dc0deecef.jpg)

باید بدانیم هرپروژه ای که هیچ سرویسی ارائه نمیدهد و محصولی ندارد و فقط میگوید من قرارداد هوشمند هستم به شدت کلاهبرداری است .

**Comment by Blockunity** [comment]
![Screenshot 2022-08-12 153012](https://user-images.githubusercontent.com/72666334/184363981-c2cc3119-92fc-47e9-a2e8-00eaf9d674cc.png)

بیشتر کاربر ها از این کشور ها هستند
